content/post/analytics_brain_trust.md

```diff
@@ -0,0 +1,86 @@
+
+
+### Step 2: Set a goal and do your research
+
+Make sure you have a defined knowledge goal before you begin reaching out to people and be as specific as possible. There is nothing worse than giving your time to talk to someone when they don't know what they want. 
+
+In my case my goal was: 
+
+>I am building an Analytics team, so I am looking to speak with others who have done it before. Specifically I would like to learn about specialist vs generalist models, titles and how the Analytics team interfaces with the rest of the organization.
+
+I also recommend doing specific research on the people you contact to understand what relevant experience or knowledge they may have. This will allow you to craft your message and, in the event of a conversation, your questions.
+
+
+### Step 3: Reach out to your warmest connections (and be accommodating)
+
+Message your warmest connections who may have relevant knowledge or may know others who have relevant knowledge. If you don't have any warm connections, contact people who you have something in common with (e.g., university, home town, industry). Linkedin is a great place to meet new contacts, but industry-specific or topic-specific messaging boards / Slack communities are even better.
+
+Craft your message so that it is specific, friendly and most importantly accommodating. Make meeting you sound like a very light lift. Here is a message I sent many years ago:
+
+>Hi XXX, I am building out an Analytics team and would love to chat with you to learn about how you've gone about building your team at YYY company. I can imagine you have similar challenges (e.g., interfacing with stakeholders and deciding on the generalist vs specialist model). Happy to work around your schedule and come to you. Let me know when works best.
+
+
+### Step 4: Ask good questions and make a genuine connection
+
+People can quickly tell if you are being disingenuous and only want a job or an introduction. I advise going into the conversation with the goal of learning and potentially forming a professional or personal relationship - a job or an introduction is icing on the cake. I met many of my closest friends in the NYC scene through conversations just like this.
+
+During the conversation, make sure you have a list of topics you want to cover, but allow the conversation to flow naturally. You don't want your coffee chat to feel like an interrogation. Also, if you force the conversation down a specific path you may miss a relevant topic that your coffee mate is especially knowledgeable or passionate about.
```

Comment:
The way I do this is i'll brainstorm questions in advance and write them down, but during the conversation I mostly keep my notebook closed. If there's a lull I might say something like "I think there was one more thing I wanted to ask ..." and then flip it open.

You need to find a balance between "stenographer" and just-passing-the-time.

Also: always write down notes immediately after the conversation!